aw-datastore/src/datastore.rs

```diff
@@ -459,7 +490,32 @@ impl DatastoreInstance {
                 )))
             }
         };
-        for event in &mut events {
+
+        'event: for event in &mut events {
```

Comment:
It should not be the datastores responsibility to ignore apps/titles imo. It should not manipulate any data. I think it's better to have this logic in the rest endpoint for insert_event and heartbeat.

Reply:
I'll have to re-look but I could have sworn there was more than just insert_event and heartbeat that could trigger the insert, which is why I was trying to consolidate the code only in two paths rather than have multiple places for the code...   But maybe I miss read the code and only insert_event and heardbeat call these two datastore code paths...